pkgs/development/python-modules/scikit-survival/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, buildPythonPackage
+, cython
+, ecos
+, fetchPypi
+, joblib
+, numexpr
+, numpy
+, osqp
+, pandas
+, pytestCheckHook
+, scikitlearn
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-survival";
+  version = "0.15.0.post0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "572c3ac6818a9d0944fc4b8176eb948051654de857e28419ecc5060bcc6fbf37";
+  };
+
+  nativeBuildInputs = [
+    cython
+  ];
+
+  propagatedBuildInputs = [
+    ecos
+    joblib
+    numexpr
+    numpy
+    osqp
+    pandas
+    scikitlearn
+    scipy
+  ];
+
+  pythonImportsCheck = [ "sksurv" ];
+
+  checkInputs = [ pytestCheckHook ];
+  pytestFlagsArray = [ "tests/" "--durations=20" ];
```

Comment:
Both of these are probably redundant in normal operation. I'm OK w/ leaving in durations if desired, but not necessary.